internal/cmd/signup/cmd.go

```diff
@@ -41,7 +41,7 @@ func Command() *cobra.Command {
 func Run() error {
 	saucebotSignup := `
                    (‾)
-                   ||                          Puppeteer,
+                   ||
```

Comment:
Just realized there is an old sauce robot here. I think we should also remove it.

Reply:
+1

Reply:
I didn't even connected the dots ! Good catch !